src/twisted/web/script.py

```diff
@@ -175,8 +181,8 @@ def render(self, request):
             io = NativeStringIO()
             traceback.print_exc(file=io)
             output = util._PRE(io.getvalue())
-            if _PY3:
-                output = output.encode("utf8")
+            if not isinstance(output, bytes):
+                output = output.encode("utf-8")
```

Comment:
The docstring for `PythonScript` should say that it requires the script to be UTF-8.

Reply:
PEP 3120 says the default encoding is UTF-8, so that is a reasonable default: https://docs.python.org/3/reference/lexical_analysis.html#lexical-analysis